src/components/atoms/typography.astro

```diff
@@ -32,7 +32,7 @@ const variantToClassName = {
   'item-title': 'text-xl font-extrabold text-gray-900',
   'item-title-suffix': 'text-xl font-medium text-gray-700',
   'item-subtitle': 'text-md font-medium text-gray-700',
-  'tile-title': 'text-sm font-medium text-gray-700',
+  'tile-title': 'text-sm font-semibold text-gray-700',
```

Comment:
Zerkając na mockup wydaje mi się, że to było dobrze
![CleanShot 2022-09-13 at 00 20 38@2x](https://user-images.githubusercontent.com/22938071/189768693-ee621669-3e33-417e-99f8-f1d22d4f1e0a.png)

Niemniej wiadomo, może zmieniłem dopiero po tym, jak zaczęłaś to zadanie 😅